### DISCUSSION TITLE
Group objects in different areas

### DISCUSSION THREAD

**Discussion by florianschendel** [netbox-community/netbox#13740] [Ideas]
We have the requirement to group different areas/zones of our networks

like DMZ, Datacenter, Campus, R&D...

It´s **NOT** a physical separation we have to implement a logical separation.
The regions are used for the geographical segmentation like north-america, south-america, emea ...
Tenancy is used to control the access rights to the objects.

My idea is to have under organization/sites a new option that is called zones

e.g.
site == physical site or subsidary. e.g hq-berlin
Within the site hq-berlin i have multiple locations (builindgs or rooms) like office building1, bulindg2 or datacenter1, datacenter2

The area DMZ is in datacenter1 and datacenter2, R&D in the building1 and buidling2.
I want to group within the areas the VRFs, IP Prefixes(addresses), vlans, devices... The same information that we actually group under sites.

**Or in other words "sub site" == area/zone**

**Comment by candlerb** [comment]
If each particular asset can only be associated with a single "zone", then I think a custom field would be the way to approach this.

If each particular asset can belong to multiple "zones", then I'd go with either tags or a multi-select custom field.

You can bind a custom field to multiple object types.  Tags apply to all object types, although in 3.7 you'll also be able to restrict them to specific object types.
